docs/learn/learn-xcm-pallet.md

```diff
@@ -10,12 +10,90 @@ slug: ../learn-xcm-pallet
 The XCM pallet ([`pallet-xcm`](https://github.com/paritytech/polkadot/tree/master/xcm/pallet-xcm))
 provides a developer-friendly interface for most of the common XCM messages.
 
-This pallet provides default implementations for many traits required by `XcmConfig`, which is
-highly useful for configuration.
+This pallet provides some default implementations for some traits required by `XcmConfig`, as well
+as an instance of the XCM Executor provided as a trait within the pallet's own configuration.
+`pallet-xcm` provides a default interface in the form of a pallet, that can manage and deal with
+XCM-related storage and higher-level dispatchable functions.
 
 It defines a set of extrinsics that can be utilized to build XCVM programs, either to target the
-local or external chains. Pallet-xcm's functionality is separated into three categories:
+local or external chains. `pallet-xcm`'s functionality is separated into three categories:
+
+:::note
+
+Remember, all XCM messages are effectively XCVM programs that contain a set of instructions. It is
```

Comment:
```suggestion
Remember, all XCMs are effectively XCVM programs that contain a set of instructions. It is
```

I know we say "XCM messages" colloquially, but do remember that the term is an oxymoron, because the M in XCM already stands for message.

Reply:
I think it would be preferable to use XCMs over XCM messages, I was debating which to use throughout.